src/scripts/data-node-install.sh

```diff
@@ -145,6 +145,9 @@ while getopts :n:m:v:A:R:K:S:Z:p:U:I:c:e:f:g:t:s:o:a:k:L:C:B:Xxyzldjh optname; d
     k) #azure storage account key for azure cloud plugin
       STORAGE_KEY="${OPTARG}"
       ;;
+    E) #azure storage account endpoint suffix
```

Comment:
Want to keep the provisioning of a storage account for azure-repository plugin separate.